build/css/live-editor.ui.css

```diff
@@ -390,35 +390,35 @@ button.hilite {
     box-shadow: 0 0 20px green;
 }
 
-/*
- * Contains styles for breakpoints and modifies the editor's appearance
- * when debugging.
- */
+dialog {
```

Comment:
Of course.  What kind of issues did you see?  How small is small?  Also, it seems like there's a different modal that's used for some things in production in addition the jquery-ui modal that's used by the live-editor itself for the picker.  It would be nice if there was some way to use the same modal code.